StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -566,7 +566,7 @@ extension STPAPIClient: FinancialConnectionsAPIClient {
             "credentials": [
                 "consumer_session_client_secret": consumerSessionClientSecret,
             ],
-            "locale": Locale.current.identifier,
+            "locale": Locale.current.toLanguageTag(),
```

Comment:
we can verify that this is expected via here:

https://github.com/stripe/stripe-ios/blob/52e9b64c/StripePaymentSheet/StripePaymentSheet/Source/Internal/API%20Bindings/Link/STPAPIClient+Link.swift#L234